x-pack/filebeat/module/salesforce/_meta/config.yml

```diff
@@ -0,0 +1,48 @@
+- module: salesforce
+  # All logs
+  login:
+    enabled: false

```

Comment:
Do you think that any of these filesets should be enabled by default? I mean, whenever user enables the filebeat module.

Reply:
That's right, I thought of the same and tried that out earlier. But I faced the following, and hence decided to go with false.
`salesforce module dataset login must be explicitly disabled (needs `enabled: false`)`